src/bootstrap/bootstrap.py

```diff
@@ -434,6 +434,12 @@ def download_toolchain(self, stage0=True, rustc_channel=None):
             with output(self.rustc_stamp(stage0)) as rust_stamp:
                 rust_stamp.write(key)
 
+            gcc_libs_hack = '{}/missing-libs-hack'.format(self.rust_root)
```

Comment:
Got this from https://github.com/msys2/MINGW-packages/blob/ad72449e4dff4fb350516c1e71737ac256543499/mingw-w64-rust/PKGBUILD#L70L78. Would switching to Clang completely eliminate the need for this hack?

Reply:
We would have to bootstrap from Clang based toolchain instead of GCC based one.